apps/base-docs/sidebars.js

```diff
@@ -4,20 +4,10 @@ module.exports = {
     ['using-base'],
     {
       type: 'category',
-      label: 'Building on Base',
+      label: 'Notices',
       collapsible: false,
       collapsed: false,
-      items: [
-        'building-with-base/network-information',
-        'building-with-base/base-contracts',
-        'building-with-base/fees',
-        'building-with-base/differences',
```

Comment:
@wbnns  did we mean to remove these?

Reply:
@anikaraghu Thanks for catching this! No, just updated and fixed. 🙏